src/main.js

```diff
@@ -24,18 +37,72 @@ render(main, getFilmsBlockTemplate());
 
 const filmList = main.querySelector(`.films-list`);
 const filmAllList = filmList.querySelector(`.films-list__container--main`);
-const filmTopRatedList = document.querySelector(`.films-list__container--top-rated`);
-const filmMostCommentedList = document.querySelector(`.films-list__container--most-commented`);
 
-const generateMarkup = (num, generator) =>
-  Array.from({length: num}, generator).join(`\n`);
+const renderCards = (arr, place, start, end) => {
+  render(place, arr.slice(start, end).map(getFilmCardTemplate).join(``));
+};
+
+const getTaskQuantityParam = () => {
+  let quantity = cardStat.leftToShow > cardDisplay.PER_PAGE ? cardDisplay.PER_PAGE : cardStat.leftToShow;
+
+  cardStat.updateTaskStat(quantity);
+  return quantity;
+};
+
+renderCards(cards, filmAllList, cardStat.quantityCounter, getTaskQuantityParam());
+
+// pop-up deatails
+
+const onCommentLinkClick = () => {
+  render(body, getFilmPopupTemplate);
```

Comment:
![toString](https://i.imgur.com/pKQaA1Q.jpg)

```suggestion
  render(body, getFilmPopupTemplate());
```